### PR TITLE
wysiwyg paste rules

### PR DESCRIPTION
**Pull request by nelsonpecora**
* allows pasting multiple paragraphs in a more consistent manner
* allows pasting an arbitrary number of different embeds, text, etc
* _super_ extendible with custom paste rules
* updated docs
* replaces component if it's empty when you paste in the embed url/etc for another component

8cddd76b-d9c9-4023-b015-314ccdea6d42-2392-00064663e6cf2f5a

4c258af8-d3ce-4c1b-a5ba-b1e9243a5d49-2392-000647677ba087ff

### CODE REVIEW

File: behaviors/wysiwyg.js [hunk 1]
```diff
@@ -159,7 +180,6 @@ function createEditor(field, buttonsWithOptions) {
         'meta',
         'script',
         'style',
-        'img',
```
Comment (nelsonpecora) [inline behaviors/wysiwyg.js]: medium-editor already removes these

File: behaviors/wysiwyg.js [hunk 2]
```diff
@@ -169,7 +189,6 @@ function createEditor(field, buttonsWithOptions) {
         // thus allowing cleanTags (above) or text-model to manage them
         [/<h[1-9]>/ig, '<h2>'],
         [/<\/h[1-9]>/ig, '</h2>'], // force all headers to the same level
-        [/<\/?blockquote(.*?)>/ig, ''], // unwrap blockquotes
```
Comment (nelsonpecora) [inline behaviors/wysiwyg.js]: we want to parse blockquote tags in our pasting rules

File: behaviors/wysiwyg.js [hunk 3]
```diff
@@ -34,63 +34,84 @@ function selectAfter(node) {
...
-  selection.addRange(range);
-  // Of the DOM, to inputte's end they wende
-  // the hooly blisful caret for to seke
-  // that hem hath holpen, when that they were pasted
+  if (node && node.nodeType === 1) {
+    range.setStartAfter(node); // set the caret after this node
+    range.collapse(true);
+    selection.removeAllRanges();
+    selection.addRange(range);
+    // Of the DOM, to inputte's end they wende
```
Comment (amycheng) [inline behaviors/wysiwyg.js]: 👌

**Comment by amycheng** [comment]
:+1: _(one day I'll learn how to use GitHub Code Reviews properly)_

**Pull request merged**